js/index.js

```diff
@@ -82,3 +82,56 @@ document.addEventListener('DOMContentLoaded', () => {
     collection.removeBook(button);
   }));
 });
+
+// website with navigation
+
+document.getElementById('list').addEventListener('click', () => {
+  document.getElementById('list_').classList.remove('d-none');
+  document.getElementById('contact_').classList.add('d-none');
+  document.getElementById('addNew').classList.add('d-none');
+});
+
+document.getElementById('add_new').addEventListener('click', () => {
+  document.getElementById('list_').classList.add('d-none');
+  document.getElementById('contact_').classList.add('d-none');
+  document.getElementById('addNew').classList.remove('d-none');
+});
+
+document.getElementById('contact').addEventListener('click', () => {
+  document.getElementById('list_').classList.add('d-none');
+  document.getElementById('contact_').classList.remove('d-none');
+  document.getElementById('addNew').classList.add('d-none');
+});
+
+// time
+function checkTime(i) {
+  if (i < 10) {
+    i = `0${i}`;
+  }
+  return i;
+}
+function startTime() {
+  const today = new Date();
+  let hr = today.getHours();
+  let min = today.getMinutes();
+  let sec = today.getSeconds();
+  hr = (hr === 0) ? 12 : hr;
+  hr = (hr > 10) ? hr + 0 : hr;
+  hr = checkTime(hr);
+  min = checkTime(min);
+  sec = checkTime(sec);
+  document.getElementById('clock').innerHTML = `${hr}:${min}:${sec}`;
+  const months = ['January', 'February', 'March', 'April', 'May', 'Jun', 'July', 'August', 'September', 'October', 'November', 'December'];
+  const days = ['Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat'];
+  const curWeekDay = days[today.getDay()];
+  const curDay = today.getDate();
+  const curMonth = months[today.getMonth()];
+  const curYear = today.getFullYear();
+  const date = `${curWeekDay},${curDay} ${curMonth} ${curYear}`;
+  document.getElementById('date').innerHTML = date;
+
+  // eslint-disable-next-line no-unused-vars
+  const time = setTimeout(() => { startTime(); }, 500);
+}
```

Comment:
- [ ] I appreciate your great effort to create this function  :clap:  However you should use luxon library to get the current date and time. Please check [the project requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m4_full_website.md#project-requirements) for more information about the luxon library.

> - Display the current date below the navigation bar. For this requirement you should use a library - [Luxon](https://moment.github.io/luxon/#/install).
>  - See the article [Get the current date and time](https://moment.github.io/luxon/#/tour?id=get-the-current-date-and-time).

Reply:
Done thanks